include/logging/log_ctrl.h

```diff
@@ -66,6 +66,8 @@ int log_set_timestamp_func(timestamp_get_t timestamp_getter, u32_t freq);
 /**
  * @brief Switch logger subsystem to panic mode.
  *
+ * Returns immediately if logger already in panic mode.
```

Comment:
Minor grammar thing: "Returns immediately if +the + logger + is + already in panic mode."

Reply:
done